doc/modularity.rst

```diff
@@ -323,9 +323,8 @@ the effects returned by the underlying reducer might need.
               return {m, lager::noop};
           },
           [&] (DocumentAction a) {
-              auto doc = static_cast<DocumentModel>(m);
               auto eff = lager::effect<DocumentAction, DocumentDeps>{lager::noop};
-              lager::invoke_reducer<DocumentDeps>(r, doc, a, [&](auto e) {
+              auto doc = lager::invoke_reducer<DocumentDeps>(r, static_cast<DocumentModel>(m), a, [&](auto e) {
```

Comment:
Just one caveat: too long lines are hard to read in the documentation and render incorrectly in many devices, it would be nice to keep it at 80 chars like the rest of the code. You can either add line breaks betwen arguments, or use named variable for `static_cast<DocumentModel>(m)`. Thanks!

Reply:
Fixed.